pkg/cnab/config-adapter/adapter.go

```
@@ -421,12 +425,18 @@ func (c *ManifestConverter) generateDependencies() *cnab.Dependencies {
 			dependencyRef.Version = &cnab.DependencyVersion{
 				Ranges: []string{dep.Bundle.Version},
 			}
+
+			// If we can detect that prereleases are used in the version, then set AllowPrereleases to true
+			v, err := semver.NewVersion(dep.Bundle.Version)
```

Should we return the error here if it's not nil?